src/node_platform.cc

```diff
@@ -524,6 +524,12 @@ std::unique_ptr<v8::JobHandle> NodePlatform::PostJob(v8::TaskPriority priority,
       this, priority, std::move(job_task), NumberOfWorkerThreads());
 }
```

Comment:
Do we still need to supply `PostJob()` since `v8::platform` already implements it which simply calls `CreateJob()`? That would help get rid of some code duplication here.

Reply:
I think there's a slight difference in implementation with CreateJob vs PostJob - CreateJob doesn't immediately schedule |worker_task| to run, whereas PostJob does. If that doesn't seem like it would be an important distinction for embedders, though, happy to remove PostJob in this PR!

Reply:
To be clear, I have no problem with `CreateJob()` and I understand the two functions serve different purposes. However, if you look at the code I linked to, v8's `PostJob()` currently calls out to `CreateJob()`, which in our implementation currently has the exact same code as in our `PostJob()`.

Reply:
hmm, looks like, `CreateJob()` isn't in V8 until https://chromium-review.googlesource.com/c/v8/v8/+/3668842 which is not yet in our main branch. So I guess this doesn't actually compile right now? (because there is nothing ot override?)

Reply:
@mscdex Oh sorry about that, I misunderstood your question (was looking at the Chromium/v8 implementation of PostJob and CreateJob). Just to add some context: Electron recently [pulled in a new roll of changes from Chromium/v8](https://github.com/electron/electron/pull/34993) and hit this error (`unimplemented pure virtual method 'CreateJob' in 'NodePlatform'`).

We wanted to upstream the fix, just so you all didn't run into a similar error later, but I think I was a bit overzealous - @joyeecheung is correct, it looks like the change that broke this for us hasn't landed in main yet. 🙂 Happy to 1) wait until those changes land in Node's main branch and re-open a PR, or 2) apply a commit similar to what targos linked above, whatever you all think is the best way forward! Also happy to close this out if it's unnecessary 🙇‍♀️